extensions/mssql/src/main.ts

```diff
@@ -137,35 +137,47 @@ export async function activate(context: vscode.ExtensionContext): Promise<MssqlE
 	azdata.ui.registerModelViewProvider('bdc-endpoints', async (view) => {
 
 		const endpointsArray: Array<Utils.IEndpoint> = Object.assign([], view.serverInfo.options['clusterEndpoints']);
+		endpointsArray.forEach(endpointInfo => {
+			endpointInfo.isHyperlink = true;
+			endpointInfo.hyperlink = 'https://' + endpointInfo.ipAddress + ':' + endpointInfo.port;
+
+		});
 		if (endpointsArray.length > 0) {
-			const managementProxyEp = endpointsArray.find(e => e.serviceName === 'management-proxy');
+			const managementProxyEp = endpointsArray.find(e => e.serviceName === 'management-proxy' || e.serviceName === 'mgmtproxy');
 			if (managementProxyEp) {
-				endpointsArray.push(getCustomEndpoint(managementProxyEp, 'Grafana Dashboard', '/grafana'));
-				endpointsArray.push(getCustomEndpoint(managementProxyEp, 'Kibana Dashboard', '/kibana'));
+				endpointsArray.push(getCustomEndpoint(managementProxyEp, 'Metrics Dashboard', '/grafana'));
```

Comment:
Please localize these endpoints since we control localization for them. For others that we get back directly from the APIs, we can't localize but these we are able to.